pkgs/applications/science/misc/golly/default.nix

```
@@ -62,13 +77,18 @@ stdenv.mkDerivation rec {
 
     mkdir -p "$out/share/golly"
     cp -r ../{Help,Patterns,Scripts,Rules} "$out/share/golly"
+
+    runHook postInstall
   '';
 
   meta = {
     description = "Cellular automata simulation program";
-    license = lib.licenses.gpl2;
-    maintainers = [ lib.maintainers.raskin ];
-    platforms = lib.platforms.linux;
+    license = lib.licenses.gpl2Plus;
+    maintainers = with lib.maintainers; [
+      raskin
+      siraben
+    ];
+    platforms = lib.platforms.unix;
     downloadPage = "https://sourceforge.net/projects/golly/files/golly";
```

```suggestion
    homepage = "https://golly.sourceforge.io/";
    downloadPage = "https://sourceforge.net/projects/golly/files/golly";
```
If you add this I can close my PR :)